### PR TITLE
GLPI 10.0 compatibility

### PR DESCRIPTION
**Pull request by cedric-anne**

### CODE REVIEW

**Comment by stonebuzz** [comment]
CSS like GLPI

image

**Comment by stonebuzz** [comment]
Login page is slightly offset (due to the use of the hook ```Hooks::DISPLAY_LOGIN```)

image

partially fixed by https://github.com/glpi-project/glpi/pull/10176

I don't have a clean solution at the moment

File: left.php [hunk 1]
```diff
@@ -35,20 +35,14 @@
...
-// Send extra expires header if configured
-echo "<meta http-equiv='Expires' content='Fri, Jun 12 1981 08:20:00 GMT'>\n";
-echo "<meta http-equiv='Pragma' content='no-cache'>\n";
-echo "<meta http-equiv='Cache-Control' content='no-cache'>\n";
-echo "<link rel='stylesheet' type='text/css' href='$treeview_url/css/treeview.css' type='text/css'>\n";
-echo "<script type='text/javascript' src='$treeview_url/js/treeview.js'>
-      </script></head>\n";
+echo "\n<html>";
+
+Html::includeHeader("TreeView");
```
Comment (cedric-anne) [inline left.php:40]: @stonebuzz  `Html::includeHeader()`  will output `<!DOCTYPE html>` and `<html>` tags and will produce an invalid DOM.

Comment (stonebuzz) [reply left.php:36]: done by removing duplicate HTML tag (from plugin)

File: inc/config.class.php [hunk 2]
```diff
@@ -574,6 +574,11 @@ public static function loginPageToTop() {
          function() {
             if (top != self)
                top.location = self.location;
+
+            div_Right = $(.col-auto.offset-md-1.p-2.text-center);
+            if (div_Right.length){
+               console.log('exist');
+            }
```
Comment (cedric-anne) [inline inc/config.class.php]: This is debug code.

**Pull request merged**